lensless/util.py

```diff
@@ -295,3 +295,103 @@ def autocorr2d(vals, pad_mode="reflect"):
 
     # remove padding
     return autocorr[shape[0] // 2 : -shape[0] // 2, shape[1] // 2 : -shape[1] // 2]
+
+
+def load_drunet(model_path, n_channels=3, requires_grad=False):
+    """
+    Load a pre-trained Drunet model.
+
+    Parameters
+    ----------
+    model_path : str
+        Path to pre-trained model.
+    n_channels : int
+        Number of channels in input image.
+    requires_grad : bool
+        Whether to require gradients for model parameters.
+
+    Returns
+    -------
+    model : :py:class:`~torch.nn.Module`
+        Loaded model.
+    """
+    from lensless.drunet.network_unet import UNetRes
+
+    model = UNetRes(
+        in_nc=n_channels + 1,
+        out_nc=n_channels,
+        nc=[64, 128, 256, 512],
+        nb=4,
+        act_mode="R",
+        downsample_mode="strideconv",
+        upsample_mode="convtranspose",
+    )
+    model.load_state_dict(torch.load(model_path), strict=True)
+    model.eval()
+    for k, v in model.named_parameters():
+        v.requires_grad = requires_grad
+
+    return model
+
+
+def apply_CWH_denoizer(model, image, noise_level=10, device="cpu", mode="inference"):
```

Comment:
denoiser

Reply:
remove "CWH" and explain in the docstrings that it expects this channel ordering?